x-pack/test/functional/services/observability/alerts/rules_page.ts

```diff
@@ -30,12 +30,24 @@ export function ObservabilityAlertsRulesProvider({ getService }: FtrProviderCont
     await find.clickByButtonText('metric-threshold');
   };
 
+  const clickOnObservabilityCategory = async () => {
+    const categories = await testSubjects.find('ruleTypeModal');
+    const category = await categories.findByCssSelector(`.euiFacetButton[title="Observability"]`);
+    await category.click();
```

Comment:
nit
```suggestion
    await category.clickByCssSelector(`.euiFacetButton[title="Observability"]`);
```

Reply:
I get the following error when using `clickByCssSelector`:
```
fail: ObservabilityApp Custom threshold rule shows the custom threshold rule in the observability section
       │      TypeError: categories.clickByCssSelector is not a function
```